spec/unit/crypto/algorithms/megolm.spec.js

```diff
@@ -144,7 +148,7 @@ describe("MegolmDecryption", function() {
                 megolmDecryption.shareKeysWithDevice(keyRequest);
 
                 // it's asynchronous, so we have to wait a bit
-                return awaitEnsureSessions;
+                return awaitEncryptForDevice;
```

Comment:
There are two steps here:
1: ensure we have olm sessions for each of the devices (this has always been asynchronous as it involves API calls)
2: encrypt the content for each device (which used to be synchronous, but is now async)

In short, we now need to wait for step 2 to happen, whereas previously waiting for step 1 was sufficient